lib/galaxy/util/tool_shed/xml_util.py

```diff
@@ -47,7 +28,7 @@ def create_element(tag, attributes=None, sub_elements=None):
     key / value pairs in the received attributes and sub_elements.
     """
     if tag:
-        elem = XmlET.Element(tag)
+        elem = etree.Element(tag)
```

Comment:
`Element = getattr(etree, '_Element', etree.Element)` ?

Reply:
This is only necessary for the `isinstance` check, since the actual `Element` class is `_Element` and `Element` is the constructor.